web/tests/functional/diff_local_remote/__init__.py

```diff
@@ -118,8 +124,7 @@ def setup_package():
         sys.exit(1)
     print('Analyzing new was successful.')
 
-    # Store results to the remote server.
-    test_project_name_remote = project_info['name'] + '_' + uuid.uuid4().hex
+    # Store results again to the remote server.
```

Comment:
Please extend the comment here too with the information that we need the second store to set the detection status to the required states.